README.md

```diff
@@ -566,6 +566,27 @@ makeRequest(req)
     .then(process.exit);
 ```
 
+Example with `async / await`:
+
+```js
+const cote = require('cote');
+const randomRequester = new cote.Requester({ name: 'Random Requester' });
+
+const req = {
+    type: 'randomRequest',
+    val: Math.floor(Math.random() * 10),
+};
+
+async function  makeRequest () {
+  const response = await randomRequester.send(req);
+  console.log(response);
+}
+
+makeRequest(req)
+    .then(process.exit);
```

Comment:
I think it's better not to mix promises with async/await. Could you put the process exit in the `makeRequest` function? Or another `async function main` that awaits `makeRequest` and then exits.

Reply:
Sure